sandbox-backend/src/main/java/com/exadel/sandbox/team5/util/CategoryWithTagWithoutIdCategory.java

```diff
@@ -0,0 +1,16 @@
+package com.exadel.sandbox.team5.util;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.util.Set;
+
+@Getter
+@Setter
+@NoArgsConstructor
+public class CategoryWithTagWithoutIdCategory {
```

Comment:
It think that `CategoryWithTagsDto` is easier

Reply:
Done